src/components/Comments/ReplyCommentItem.js

```diff
@@ -314,11 +373,18 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        {<Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
         <TouchableOpacity onPress={onUpVote} testID="upvotebtn" activeOpacity={1}>
           <ButtonHightlight
             onLongPress={handleLongPress}
-            style={[styles.arrowdown, styles.btn]}
+            style={[
+              styles.arrowdown,
+              styles.btn,
+              {
+                paddingLeft: 0,
+                marginLeft: 8
+              }
+            ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition check for `totalVote` has been removed in the new hunk. This could potentially lead to displaying a "0" vote count, which might not be desirable from a UX perspective. If you don't want to display a "0" vote count, consider adding the condition back.
```diff
        {<Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        {totalVote !== 0 && <Text style={styles.vote(voteStyle())>{totalVote}</Text>}
         <TouchableOpacity onPress={onUpVote} testID="upvotebtn" activeOpacity={1}>
           <ButtonHightlight
             onLongPress={handleLongPress}
            style={[
              styles.arrowdown,
              styles.btn,
              {
                paddingLeft: 0,
                marginLeft: 8
              }
            ]}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->